### ISSUE TITLE
shop = pest_control

### ISSUE THREAD

**Issue by matkoniecz** [openstreetmap/id-tagging-schema#1211]
### OSM Tag(s)

shop = pest_control

### How would you like this tag to see supported?

As a Preset

### Label

Pest control goods shop (???)

### Terms

rodents, rats, mice, insects

### Link to OSM Wiki page

https://wiki.openstreetmap.org/wiki/Tag%3Ashop%3Dpest_control

### Status of the Tag

In Use

### Usage of the tag

218

### Replaces other Tag?

no

### Further Information

It is used by NSI presets and seems to make sense as a distinct shop value

**Comment by pandadeepimpact** [comment]
I agree with the label "Pest Control Goods Shop". Weird as it may sound, having the word "Goods" makes it clear that it's a business that sells items, not services (as opposed to `craft=pest_control`).

**Comment by matkoniecz** [comment]
For icon: I guess that `    "icon": "fas-bugs",` would work well

https://github.com/openstreetmap/id-tagging-schema/blob/339ae5bd55f0c0e8063fb21641eab3ad85b3fa85/data/presets/man_made/insect_hotel.json#L14

**Comment by matkoniecz** [comment]
So ideally we would have:

`shop = pest_control` Pest Control Goods Shop
`craft = pest_control` Pest Control Services Shop
`shop = pest_control craft = pest_control` Pest Control Goods and Services Shop

is the last one possible in iD presets?

**Comment by matkoniecz** [comment]
though maybe we should get `craft = pest_control` deprecated: how often you have places doing only one of these?

Is it really worth distinction in top-level tag?

**Comment by lumikeiju** [comment]
Stats as of 2026-08-22:
306 `shop=pest_control` [Taginfo](https://taginfo.openstreetmap.org/tags/shop=pest_control#overview)
186 `craft=pest_control` [Taginfo](https://taginfo.openstreetmap.org/tags/craft=pest_control#overview)
4 `shop=pest_control`+`craft=pest_control` [Overpass Turbo](https://overpass-turbo.eu/s/2viR)

**Comment by Dimitar5555** [comment]
> though maybe we should get craft = pest_control deprecated: how often you have places doing only one of these?

Quite common actually. Professional-grade pest control often involves hazardous chemicals that require full personal protective equipment (PPE) and specialised application tools, which legally or practically cannot be sold for direct home use by consumers.

Meanwhile, general-use consumer products (basic traps, sprays, poison baits, etc.) are already sold in agrarian shops, hardware stores, garden centers, pharmacies, supermarkets, etc. rather than dedicated standalone pest control retail stores.

Looking at a few (about 10) of the current `shop=pest_control` which have websites tagged, it seems like they are mostly offering pest control services and not goods.

Having both presets is a good idea, but I must admit that I haven't seen a shop which only sells pest control goods. I guess that it should have lower priority at the very least.
